events/wmt08.md

```diff
@@ -89,68 +89,40 @@ The **Third Workshop on Machine Translation** (**WMT08**) took place on 19 June,
 
 Full results of the shared tasks: [*Further Meta-Evaluation of Machine Translation*](https://www.statmt.org/wmt08/pdf/WMT09.pdf)
 
-### News translation
+### Translation
 
-The results were determined with a [relative ranking](wmt.md#relative-ranking), the `≥ others` ("greater than or equal to others") score.
+The results were determined with a [relative ranking](wmt.md#relative-ranking), the `≥ others` (“greater than or equal to others”) score, [constituent ranking](wmt.md#constituent-ranking), and [yes or no constituent judgement](wmt.md#yes-or-no-constituent-judgement).
```

Comment:
I would add the determiners "the" and "a".

```
[...] the `≥ others` (“greater than or equal to others”) score, the [constituent ranking](wmt.md#constituent-ranking), and a [yes or no constituent judgement](wmt.md#yes-or-no-constituent-judgement).
```

Reply:
done